plugins/UserAgreementPlugin/UserAgreement.qml

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 1.3
+
+import UM 1.3 as UM
+
+UM.Dialog
+{
+    id: baseDialog
+    minimumWidth: Math.floor(UM.Theme.getSize("modal_window_minimum").width * 0.75)
+    minimumHeight: Math.floor(UM.Theme.getSize("modal_window_minimum").height * 0.5)
+    width: minimumWidth
+    height: minimumHeight
+    title: "User Agreement"
+
+    TextArea
```

Comment:
Is there a way to prevent the user from modifying the text in the screen?

Reply:
'readOnly: true;' is the only thing I can think of, but that is already in there